README.md

```diff
@@ -29,104 +30,143 @@ applications. More information about Thread can be found on
 
+Other features and enhancements include:
 
-## What's included ##
+* IPv6 configuration and raw data interface
+* UDP sockets
+* Application support and services
+    * CoAP client and server
+    * DHCPv6 client and server
+    * DNSv6 client
+    * Command Line Interface (CLI)
+* NCP support:
+    * Spinel - general purpose NCP protocol
+    * `wpantund` - user-space NCP network interface driver/daemon
+    * Sniffer support via NCP Spinel nodes
 
-In the repo you'll find the following directories and files:
+### What's coming?
 
-File/Folder   | Provides
---------------|----------------------------------------------------------------
-`doc`         | Code and protocol docs
-`etc`         | Configuration files for other build systems (e.g. Visual Studio)
-`examples`    | Sample applications and platforms demonstrating OpenThread
-`include`     | Public API header files
-`src`         | Core implementation of the Thread standard and related add-ons
-`tests`       | Unit and Thread conformance tests
-`third_party` | Third-party code used by OpenThread
-`tools`       | Helpful utilities related to the OpenThread project
+The development of OpenThread is ongoing to regularly provide additional features not available in the standard. Currently, we are working on:
+
+* Host-side Border Agent support
+
+# Who supports OpenThread?
+
+Led by Nest, the following companies are contributing to the ongoing development of OpenThread:
+
+<a href="https://www.arm.com/"><img src="doc/images/ot-contrib-arm.png" alt="ARM" width="200px"></a><a href="http://www.atmel.com/"><img src="doc/images/ot-contrib-atmel.png" alt="Atmel" width="200px"></a><a href="ttp://www.dialog-semiconductor.com/"><img src="doc/images/ot-contrib-dialog.png" alt="Dialog" width="200px"></a><a href="https://www.microsoft.com/en-us/"><img src="doc/images/ot-contrib-ms.png" alt="Microsoft" width="200px"></a><a href="https://nest.com/"><img src="doc/images/ot-contrib-nest.png" alt="Nest" width="200px"></a><a href="http://www.nordicsemi.com/"><img src="doc/images/ot-contrib-nordic.png" alt="Nordic" width="200px"></a><a href="https://www.qualcomm.com/"><img src="doc/images/ot-contrib-qc.png" alt="Qualcomm" width="200px"></a><a href="https://www.ti.com/"><img src="doc/images/ot-contrib-ti.png" alt="Texas Instruments" width="200px"></a>
+
+OpenThread has been ported to several devices and platforms. Build examples are included in the OpenThread library.
+
+### IEEE 802.15.4 Platform Support
 
+* [Dialog Semiconductor DA15000 OpenThread Sandbox Development Kit](https://github.com/openthread/openthread/wiki/Platforms#dialog-da15000)
+* [Nordic Semiconductor nRF52840 SoC](https://github.com/openthread/openthread/wiki/Platforms#nordic-semiconductor-nrf52840)
+* [Texas Instruments SimpleLink CC2538](https://github.com/openthread/openthread/wiki/Platforms#texas-instruments-cc2538)
+* [Texas Instruments SimpleLink CC2650](https://github.com/openthread/openthread/wiki/Platforms#texas-instruments-cc2650)
```

Comment:
I think we can simplify the above to just include the chip name:

- Dialog Semiconductor DA15000
- Nordic Semiconductor nRF52840
- Texas Instruments CC2538
- Texas Instruments CC2650